packages/conventional-commit-lint/src/index.d.ts

```diff
@@ -0,0 +1,24 @@
+// define types for a few modules used by probot that do not have their
+// own definitions published. Before taking this step, folks should first
+// check whether type bindings are already published.
+declare module "promise-events" {
```

Comment:
huh - where's that used?

Reply:
```bah
node_modules/probot/lib/application.d.ts:5:30 - error TS7016: Could not find a declaration file for module 'promise-events'. '/usr/local/google/home/bencoe/google/repo-automation-bots/packages/conventional-commit-lint/node_modules/promise-events/emitter.js' implicitly has an 'any' type.
  Try `npm install @types/promise-events` if it exists or add a new declaration (.d.ts) file containing `declare module 'promise-events';`
```

Seems to be a deep dependency of probot, we should keep an eye on the probot module itself and see if they eventually resolve this.